src/utils/Pausable.huff

```diff
@@ -0,0 +1,93 @@
+/// @title Pausable
+/// @notice SPDX-License-Identifier: MIT
+/// @author zarf.eth <https://github.com/zarfsec>
+/// @notice A Pausable implementation
+
+
+// Interface
+
+/// @notice returns whether the contract is paused
+#define function isPaused() view returns (bool)
+
+/// @notice Pauses the contract 
+/// @dev Only callable when the contract is unpaused.
+#define function pause() nonpayable returns ()
+
+/// @notice Unpauses the contract 
+/// @dev Only callable when the contract is paused.
+#define function unpause() nonpayable returns ()
+
+/// @notice Emitted when contract is paused.
+#define event Paused(address)
+
+/// @notice Emitted when contract is unpaused.
+#define event Unpaused(address)
+
+
+
+// Storage
+
+/// @notice Paused Storage Slot
+#define constant PAUSED_SLOT = FREE_STORAGE_POINTER()
+
+/// @notice Unpaused representation
+#define constant _UNPAUSED = 0x01
```

Comment:
Why do you put a _ before the constant ? Where does this convention comes from ?

By the way, I would have called it `_NOT_PAUSED` instead, because I think that "unpause" refers more to the action of removing the pause.

Reply:
agree w both comments, if `_` is supposed to be internal, then why doesnt PAUSED_LOT have it?  lets remove preceding underscore and update naming convention as proposed then merge!

Reply:
Ah my bad :) The preceding _ was an artefact from the fact the constant was first named PAUSABLE_PAUSED and PAUSABLE_UNPAUSE. I agree to remove the preceding _.

Reply:
NOT_PAUSED sounds better than UNPAUSED indeed.

Reply:
Made new pull request with the proper naming convention in the dev branch.